pom.xml

```diff
@@ -290,7 +290,7 @@
   </profiles>
 
   <properties>
-    <frontend.version>7e8d755d2538e124147ad983f4caa38ef0992f6b</frontend.version>
+    <frontend.version>3784d650dfec831b5fd4b57a206801d9e2513cd5</frontend.version>
```

Comment:
@alisman looks good but the tag is slightly funky in the frontend repo (`untagged-xxx` instead of `v3.7.5`): https://github.com/cBioPortal/cbioportal-frontend/releases/tag/untagged-903598df451f1a8f9c7d

<img width="536" alt="Screen Shot 2021-09-13 at 4 47 06 PM" src="https://user-images.githubusercontent.com/1334004/133154133-a1572bd1-5af5-4948-82e5-2438fe8b9528.png">

Reply:
fixed it. thanks